monitoring/precise_code_intel_api_server.go

```diff
@@ -36,23 +36,55 @@ func PreciseCodeIntelAPIServer() *Container {
 							Name:        "99th_percentile_db_duration",
 							Description: "99th percentile successful database query duration over 5m",
 							// TODO(efritz) - ensure these exclude error durations
-							Query:             `histogram_quantile(0.99, sum by (le,op)(rate(src_code_intel_db_duration_seconds_bucket{job="precise-code-intel-api-server"}[5m])))`,
+							Query:             `histogram_quantile(0.99, sum by (le)(rate(src_code_intel_db_duration_seconds_bucket{job="precise-code-intel-api-server"}[5m])))`,
 							DataMayNotExist:   true,
 							DataMayBeNaN:      true,
 							Warning:           Alert{GreaterOrEqual: 20},
-							PanelOptions:      PanelOptions().LegendFormat("{{op}}").Unit(Seconds),
+							PanelOptions:      PanelOptions().LegendFormat("db operation").Unit(Seconds),
 							PossibleSolutions: "none",
 						},
 						{
 							Name:              "db_errors",
 							Description:       "database errors every 5m",
-							Query:             `sum by (op)(increase(src_code_intel_db_errors_total{job="precise-code-intel-api-server"}[5m]))`,
+							Query:             `increase(src_code_intel_db_errors_total{job="precise-code-intel-api-server"}[5m])`,
 							DataMayNotExist:   true,
 							Warning:           Alert{GreaterOrEqual: 20},
-							PanelOptions:      PanelOptions().LegendFormat("{{op}}"),
+							PanelOptions:      PanelOptions().LegendFormat("db operation"),
 							PossibleSolutions: "none",
 						},
 					},
+				},
+			},
+			{
+				Title:  "Janitor",
+				Hidden: true,
+				Rows: []Row{
+					{
+						{
+							Name:              "janitor_errors",
+							Description:       "janitor errors every 5m",
+							Query:             `sum(increase(src_api_server_janitor_errors_total[5m]))`,
+							DataMayNotExist:   true,
+							Warning:           Alert{GreaterOrEqual: 20},
+							PanelOptions:      PanelOptions().LegendFormat("errors"),
+							PossibleSolutions: "none",
+						},
+						{
+							Name:              "janitor_uploads_without_bundle_files",
+							Description:       "upload records removed (due to missing bundle files) every 5m",
+							Query:             `sum(increase(src_api_server_janitor_upload_records_removed_total[5m]))`,
+							DataMayNotExist:   true,
+							Warning:           Alert{GreaterOrEqual: 20},
+							PanelOptions:      PanelOptions().LegendFormat("records removed"),
+							PossibleSolutions: "none",
+						},
+					},
+				},
+			},
+			{
+				Title:  "External services",
```

Comment:
```suggestion
				Title:  "Internal service requests",
```

To match frontend and explain these are 'internal to Sourcegraph' as a whole:

![image](https://user-images.githubusercontent.com/3173176/81976856-00632f80-95de-11ea-835b-7b045b9f928d.png)